src/services/PluginService.ts

```diff
@@ -1,5 +1,8 @@
-import { apiClient, tryRequest } from '@/api'
+
 import type { JSONSettings } from '@models/JSONSchema'
+import { useApiClient } from '../composables/useApiClient'
+
+const { apiClient, tryRequest } = useApiClient()
```

Comment:
wrong usage, you should not use composables outside of Vue components or Pinia stores.